src/table/utils/styling-utils.js

```diff
@@ -60,8 +60,14 @@ export const isUnset = (prop) => !prop || JSON.stringify(prop) === JSON.stringif
 
 export function getHeadStyle(layout, theme) {
   const header = layout.components?.[0]?.header;
-
-  return getBaseStyling(header, 'header', theme);
+  const headStyle = getBaseStyling(header, 'header', theme);
+  const setBackgroundColor = isDarkColor(headStyle.color) ? '#FAFAFA' : '#323232';
```

Comment:
rename to `backgroundColor`, this sounds like a setter, not the value of the background Color

Reply:
yup